.circleci/config.yml

```diff
@@ -19,13 +19,14 @@ jobs:
       - image: circleci/mysql:5.7
         environment:
           - MYSQL_USER=root
-          - MYSQL_ROOT_PASSWORD=ubuntu
+          - MYSQL_ROOT_PASSWORD=
```

Comment:
This makes me uncomfortable, no root PWD?

Reply:
I don't think it needs one for a temporary image.